subprojects/docs/src/docs/release/notes.md

```diff
@@ -212,6 +212,16 @@ projects like those using Spring to have correct version comparison.
 
 See [gradle/gradle#1378](https://github.com/gradle/gradle/issues/1378) for more details.
 
+### Version of Checkstyle has been upgraded
+
+By default, Gradle now uses [Checkstyle 6.19](http://checkstyle.sourceforge.net/releasenotes.html#Release_6.19). Previously, Gradle used Checkstyle 5.9.
```

Comment:
Should maybe mention why we didn't upgrade to the latest version of Checkstyle so they understand.